tests/pipelines/deepfloyd_if/test_if_superresolution.py

```diff
@@ -57,6 +57,15 @@ def get_dummy_inputs(self, device, seed=0):
 
         return inputs
 
+    def test_incorrect_input_size(self):
+        # Put an image non-divisible by 8 into the pipeline and check that it throws an Exception.
+        image = floats_tensor((1, 3, 31, 31), rng=random.Random(0)).to(torch_device)
+        generator = torch.Generator(device="cpu").manual_seed(0)
+        with self.assertRaises(ValueError):
+            self.pipeline(
```

Comment:
can we make sure this test works?

Reply:
i can't run the test suite locally, i was waiting for it to run on the workflow here

Reply:
oh so these are the tests that failed https://github.com/huggingface/diffusers/actions/runs/9044175339/job/24855064736#step:7:15620
I can trigger them again now but I think the results would be the same